runatlantis.io/docs/server-configuration.md

```diff
@@ -47,6 +47,18 @@ Values are chosen in this order:
 
 
 ## Flags
+### `--allow-commands`
+  ```bash
+  atlantis server --allow-commands=version,plan,apply,unlock,approve_policies
+  # or
+  ATLANTIS_ALLOW_COMMANDS='version,plan,apply,unlock,approve_policies'
+  ```
+  List of allowed commands to be run on the Atlantis server, Defaults to `version,plan,apply,unlock,approve_policies`
+
+  Notes:
+  * Accepts a comma separated list, ex. `command1,command2`.
+  * `version`, `plan`, `apply`, `unlock`, `approve_policies` and `import` are available.
```

Comment:
What do you think about an `all` string to capture all the commands? Or would that be better in a separate flag like `--allow-all-commands`?

Reply:
Sounds good. But when providing new command, it may become unintentional risk on some situations for someone who uses `all`. I don't need it so much.

Reply:
the allow-all-commands (or similar) is a flag i'd prefer if we expand on command availability. otherwise each new version, people who want all the commands will have to manually keep updating the --allow-commands csv string

Reply:
I see. To use new feature for Atlantis, it might be better to introduce it. I added `all` keyword 👌 

https://github.com/runatlantis/atlantis/pull/2877/commits/4bc72cadf4afc1034157c5ef60451e9eeeb427b3

Reply:
Thank you!